readme.md

```
@@ -197,7 +221,7 @@ To use on `$name` param as string.
 We have a simple docker file to test locally, run:
 
 ```sh
-$ docker-compose up
+$ docker compose up -d
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)